modulo_lib_test.cc

```diff
@@ -27,13 +27,29 @@ std::string status(bool passed) {
 }
 
 
+struct TestInputs {
+	int a, b, mod;
+};
+
 bool test_modulo_multiply() {
-    // TODO(tomichaw): Implement this.
+	TestInputs inputs[] = { {1, 2, 3}, {2, 2, 3}, {2, 2, 4} };
```

Comment:
It would be more readable if you put each input in its own line. Same for below.

Reply:
Done.

Reply:
Done.